include/api/types.h

```diff
@@ -20,6 +20,12 @@
 /* seL4_CapRights_t defined in mode/api/shared_types.bf */
 
 typedef word_t prio_t;
+
+/* The userland API uses ticks_t, which is micro seconds, while the kernel uses
```

Comment:
this comment is a bit confusing - I had to read it multiple times to understand it - and I wrote the api.

Did you consider something simpler eg:

```
ticks_t: time in ticks at the frequency of the hardware timer
time_t: time in microseconds
```

if you really think it helps, you could add

```
User level api functions all use `time_t`.  
```

Reply:
I have reworded it, what do you think now?